### PR TITLE
Add Chia Network icon

### PR DESCRIPTION
**Pull request by roybotbot**
**Issue:** n/a

**Similarweb rank:**
315,725

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- Organization Github stars total: GitHub Org's stars
- Chia Network icon
- Hex value is the official brand color
- Logo was always a vector

#### Preview
chianetwork

### Note

I'm submitting this as a Chia Network employee, where I am the Ecosystem Operations Manager.

### CODE REVIEW

**Review by dirien — COMMENTED** [review]
Hi @roybotbot,

is this the source of your icon? https://docs.chia.net/svg/chia-leaf-green.svg

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2073,6 +2073,22 @@
...
+            "license": {
+                "type": "custom",
+                "url": "https://www.chia.net/wp-content/uploads/2022/08/Chia-Trademark-Policy.pdf"
+            },
+            "aliases": {
+                "aka": [
+                    "Chia",
+                    "Chia Blockchain"
+                ]
+            }
```
Comment (LitoMore) [inline _data/simple-icons.json:2090]: The `Chia Network` is enough in this case. You don't need `aliases`.

**Comment by LitoMore** [comment]
CleanShot 2023-05-25 at 02 59 29@2x

@roybotbot I picked the https://github.com/Chia-Network/chia-blockchain-gui/blob/035b59ee61c65fb932420213f08c7aea2138d97a/packages/gui/src/assets/img/chia_circle.svg as a reference, but I found some differences.

**Review by LitoMore — APPROVED** [review]
image

@roybotbot Perfect fit! Thank you for contributing!

**Pull request merged**